pom.xml

```diff
@@ -28,7 +28,7 @@
 		<micrometer.version>1.12.1</micrometer.version>
 		<argLine /> <!-- add empty default argLine so Surefire won't fail when JaCoCo isn't present -->
 		<!-- property [iaf.rootdir] is available to get the root of the project, because of the [directory-maven-plugin] -->
-		<frankdoc.version>1.2-SNAPSHOT</frankdoc.version>
+		<frankdoc.version>2.0-SNAPSHOT</frankdoc.version>
```

Comment:
Ow, ik dacht dat we deze al eerder gebruikten, maar heb dit idd niet zelf veranderd. Volgens mij staat dit verder los van Java 17. Het draait al vanaf v8.0 met Java 11.